src/tribol/geom/GeomUtilities.cpp

```diff
@@ -760,10 +759,10 @@ void Intersection2DPolygon( const real* const RESTRICT xA,
          // if both segments are not defined by nodes interior to the other polygon
          if (checkA && checkB) 
          {
-            // debug
             if (interId >= maxSegInter) 
             {
-               SLIC_ERROR ("Intersection2DPolygon: number of segment intersections exceeds precomputed maximum.");
+               SLIC_INFO("Intersection2DPolygon: number of segment intersections exceeds precomputed maximum.");
```

Comment:
Gut-check: Does this need to be a `SLIC_INFO` and, if so, could the message be improved?

Reply:
I changed to `SLIC_DEBUG`. I think the message is useful, but not at the INFO level. I also improved the message.